src/helm/proxy/clients/openai_client.py

```diff
@@ -43,25 +41,17 @@ def __init__(
         cache_config: CacheConfig,
         api_key: Optional[str] = None,
         org_id: Optional[str] = None,
+        base_url: Optional[str] = None,
```

Comment:
Is this `base_url` set anywhere or should this be specified somewhere? Since in old requests, the URL was wet by default to `https://api.openai.com/v1`, I am wondering if old requests that are not explicitly this will now fail? Or is the behaviour of `OpenAI` to set it as this if it is None?